Dockerfile

```diff
@@ -21,11 +21,9 @@ ENV GO111MODULE=off
 FROM base AS criu
 ARG DEBIAN_FRONTEND
 ADD --chmod=0644 https://download.opensuse.org/repositories/devel:/tools:/criu/Debian_10/Release.key /etc/apt/trusted.gpg.d/criu.gpg.asc
-# FIXME: temporarily doing a manual chmod as workaround for https://github.com/moby/buildkit/issues/2114
 RUN --mount=type=cache,sharing=locked,id=moby-criu-aptlib,target=/var/lib/apt \
     --mount=type=cache,sharing=locked,id=moby-criu-aptcache,target=/var/cache/apt \
-        chmod 0644 /etc/apt/trusted.gpg.d/criu.gpg.asc \
-        && echo 'deb https://download.opensuse.org/repositories/devel:/tools:/criu/Debian_10/ /' > /etc/apt/sources.list.d/criu.list \
+        echo 'deb https://download.opensuse.org/repositories/devel:/tools:/criu/Debian_10/ /' > /etc/apt/sources.list.d/criu.list \
```

Comment:
Should we add something like `TODO update to Debian_11` or just not worry about it?  (It's probably fine.)

Reply:
I'm hoping @kolyshkin will remind me at that time; https://github.com/opencontainers/runc/pull/3171#discussion_r692213776 😁